src/power-bi/Governance.Report/definition/pages/ReportSection04cb7247170034c13d74/visuals/1e99ea437e1b8b8cd1a8/visual.json

```diff
@@ -0,0 +1,97 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/fabric/item/report/definition/visualContainer/1.1.0/schema.json",
+  "name": "1e99ea437e1b8b8cd1a8",
+  "position": {
+    "x": 15.223880597014926,
+    "y": 111.94029850746269,
+    "z": 1000,
+    "height": 194.32835820895522,
+    "width": 1232.2388059701493,
+    "tabOrder": 4000
+  },
+  "visual": {
+    "visualType": "textbox",
+    "objects": {
+      "general": [
+        {
+          "properties": {
+            "paragraphs": [
+              {
+                "textRuns": [
+                  {
+                    "value": "The "
+                  },
+                  {
+                    "value": "Governance report",
```

Comment:
Should we set the display name in the report to "Cloud policy and governance" to align with the Framework? Just a thought. I'd keep the file name simple either way. Maybe that's more confusing and not worth it. Curious what others think.